### ISSUE TITLE
Skipping backpack_flags in Browser

### ISSUE THREAD

**Issue by dani2191** [Laravel-Backpack/BackupManager#66]
# Bug report
When I click the button the backup process is performed but it skips the '--only-db' flag I've set up in backup.php.

### What I did:
I set the backpack_flags like this
'backpack_flags' => [ '--disable-notifications'=> true, '--only-db'=> true ],

### What I expected to happen:
The backup should only include the DB SQL Dump.

### What happened:
The backup includes both application and DB SQL Dump.

### What I've already tried to fix it:
I've tried the following:
'backpack_flags' => [ '--disable-notifications'=> true, '--only-db']
'backpack_flags' => [ '--only-db', '--disable-notifications'=> true]
'backpack_flags' => [ '--only-db'=> true, '--disable-notifications'=> true]
'backpack_flags' => [ '--only-db'=> true]
'backpack_flags' => [ '--only-db']

It works as expected when I run it through artisan with:
php artisan backup:run --only-db

### Backpack, Laravel, PHP, DB version:
Backpack: 3.6
Laravel: 5.8
PHP: 7.1.3
DB Version: MySQL 5.7.24

**Comment by pxpm** [comment]
You might have your config cached, so changes you made after are not taken into account. 

Please try `php artisan config:cache` to clear the cache and rebuild it from the current values.

BR,

**Comment by dani2191** [comment]
> You might have your config cached, so changes you made after are not taken into account.
> 
> Please try `php artisan config:cache` to clear the cache and rebuild it from the current values.
> 
> BR,

Tried that, and it's the same.
However, after checking the create() function in BackupController.php, when the artisan run function is called it looks like this:
<code>
public function create()
    {
        try {
            ini_set('max_execution_time', 600);

            Log::info('Backpack\BackupManager -- Called backup:run from admin interface');

            Artisan::call('backup:run');

            Log::info("Backpack\BackupManager -- backup process has started");
        } catch (Exception $e) {
            Response::make($e->getMessage(), 500);
        }

        return 'success';
    }
</code>
So I changed <code>Artisan::call('backup:run');</code> to <code>Artisan::call('backup:run --only-db');</code>
And now it works like I want to.

**Comment by xlstefan** [comment]
Same issue here.

btw off topic but also can have connection with some of the issues, backup.php config is not aligned to the original file so I created a PR for it.
Link to PR: https://github.com/Laravel-Backpack/BackupManager/pull/67
My second issue besides the original(as yours) was that I had issue that my backups where cleaned up before they should have been.

I updated the backup.conf with original content from spatie-backup.
